src/app/zap-templates/common/ChipTypesHelper.js

```diff
@@ -15,34 +15,33 @@
  *    limitations under the License.
  */
 
-function asBasicType(type)
-{
+function asBasicType(type) {
```

Comment:
Why all the changes to this function other than adding FabricId?

Reply:
There is a bug where I need to do this... something is not right in generated code

Reply:
Something is reformatting my file in vscode when I save json files. Will revert.